docs/index.md

```diff
@@ -449,9 +447,73 @@ setTimeout(function() {
 }, 5000);
 ```
 
+### Failing Hooks
+
+Upon a failing "before all", "before each" or "after each" hook, all remaining tests in the current suite and also its nested suites will be skipped. Skipped tests are included in the test results and marked as **skipped**. A skipped test is not considered a failed test.
+
+```js
+describe('outer', function() {
+  before(function() {
+    throw new Error('Exception in before hook');
+  });
+
+  it('should skip this outer test', function() {
+    // will be skipped and listed as 'skipped'
+  });
+
+  after(function() {
+    // will be executed
+  });
+
+  describe('inner', function() {
+    before(function() {
+      // will be skipped
+    });
+
+    it('should skip this inner test', function() {
+      // will be skipped and listed as 'skipped'
+    });
+
+    after(function() {
+      // will be skipped
+    });
+  });
+});
+```
+
+#### "before each" hook
+
+```js
+describe('failing "before each" hook', function() {
+  beforeEach(function() {
+    // runs and fails only once
+    throw new Error('Exception in beforeEach hook');
+  });
+
+  it('should skip this outer test', function() {
+    // will be skipped and reported as 'skipped' test
+  });
+
+  afterEach(function() {
+    /* will be executed */
+  });
+  after(function() {
+    /* will be executed */
+  });
+
+  describe('inner suite', function() {
+    // all hooks, tests and nested suites will be skipped
+
+    it('should skip this inner test', function() {
+      // will be skipped and reported as 'skipped' test
+    });
+  });
+});
+```
+
 ## Pending Tests
 
-"Pending"--as in "someone should write these test cases eventually"--test-cases are simply those _without_ a callback:
+"Pending" - as in "someone should write these test cases eventually" - test-cases are simply those _without_ a callback:
```

Comment:
Is this an "EM DASH"?  Ideally we should have something like smartypants that translates `--` to `—`.  Maybe revert this?